docker/terra-node-base/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:dubnium-alpine
+
+EXPOSE 8080
+
+ENV CI true
+
+RUN npm config set unsafe-perm true -g
+
+WORKDIR /opt/module
```

Comment:
Variable work directories is no longer available or needed. (This was originally added for terra dev site, but is no longer needed).